app/views/admin/success.njk

```diff
@@ -33,6 +33,27 @@
       <p class="govuk-body">{{ paragraph }}</p>
       {% endfor %}
 
+      {% if model.html %}
```

Comment:
Do we need to be careful of code injection here? I guess it's admin users, and the WAF should block major issues